internal/nodes/avi_model_l4_translator.go

```diff
@@ -60,7 +60,13 @@ func (o *AviObjectGraph) ConstructAviL4VsNode(svcObj *corev1.Service, key string
 			fqdns = append(fqdns, fqdn)
 		}
 	}
+	// Default: Traffic is enabled
+	// Not using it in Shared VIP as we don't have use case for it.
+	trafficEnabled := proto.Bool(true)
 
+	if traffic_disabled, ok := svcObj.Annotations[lib.VSTrafficDisabled]; ok && strings.ToLower(traffic_disabled) == "true" {
```

Comment:
Not required as Use case VCF and in VCF, it is always frontended by Gateway so workflow is different.